config/config.go

```diff
@@ -45,6 +50,8 @@ func ConfigureOptions(fs *flag.FlagSet, args []string) (*Config, error) {
 	fs.StringVar(&opts.Addr, "addr", "", "The URL or the IP address to run tests against")
 	fs.StringVar(&opts.Output, "o", "plaintext", "The type of the output")
 	fs.StringVar(&opts.Output, "out", "plaintext", "The type of the output")
+	fs.IntVar(&opts.PingCount, "p", 3, "Number of ping packets")
+	fs.IntVar(&opts.Timeout, "t", -1, "Give up on ping after this many seconds")
```

Comment:
The default variable causes panic in the pipeline, can you update the variable or update the tests?

```
=== RUN   TestRunAllTests
panic: non-positive interval for NewTicker

goroutine 9 [running]:
time.NewTicker(0x12dd865)
	/Users/runner/hostedtoolcache/go/1.17.3/x64/src/time/tick.go:24 +0x10f
github.com/go-ping/ping.(*Pinger).runLoop(0xc00014a180, {0x1349320, 0xc00004f110}, 0xc000076540)
	/Users/runner/go/pkg/mod/github.com/go-ping/ping@v0.0.0-20211014180314-6e2b003bffdd/ping.go:449 +0x9d
github.com/go-ping/ping.(*Pinger).run.func2()
	/Users/runner/go/pkg/mod/github.com/go-ping/ping@v0.0.0-20211014180314-6e2b003bffdd/ping.go:434 +0x6a
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/Users/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x67
created by golang.org/x/sync/errgroup.(*Group).Go
	/Users/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x92
```

Caused by:

https://github.com/ycd/dstp/blob/4791e051bbe9ad876fef6d2f24ae4a676b133d43/pkg/dstp/dstp_test.go#L11-L19